gradle/libs.versions.toml

```diff
@@ -34,6 +36,8 @@ lifecycleRuntimeCompose = "2.7.0"
 
 [libraries]
 accompanist-permissions = { module = "com.google.accompanist:accompanist-permissions", version.ref = "accompanist" }
+androidx-connect-client = { module = "androidx.health.connect:connect-client", version.ref = "connectClient" }
```

Comment:
Dependency alias 'androidx-connect-client' is not used in build scripts. Remove this.